packages/manager/apps/account-creation/src/context/tracking/tracking.provider.spec.tsx

```diff
@@ -0,0 +1,286 @@
+import { describe, expect, it, vi, beforeEach } from 'vitest';
+import { render, screen, waitFor } from '@testing-library/react';
+import { TrackingPlugin } from '@ovh-ux/shell';
+import { Region, User } from '@ovh-ux/manager-config';
+import * as shellClient from '@ovh-ux/manager-react-shell-client';
+import { TrackingProvider } from './tracking.provider';
+import { APP_NAME, LEVEL2, SUB_UNIVERSE, UNIVERSE } from '@/tracking.constant';
+import { useTrackingContext } from './useTracking';
+
+const mockConfigureTracking = vi.fn();
+// Mock dependencies
+vi.mock('@ovh-ux/shell', () => ({
+  TrackingPlugin: vi.fn().mockImplementation(() => ({
+    configureTracking: mockConfigureTracking,
+    trackPage: vi.fn(),
+    trackClick: vi.fn(),
+  })),
+}));
+
+vi.mock('@ovh-ux/manager-react-shell-client', () => ({
+  getClickProps: vi.fn((params) => params),
+  getPageProps: vi.fn((params) => params),
+}));
+
+describe('TrackingProvider', () => {
+  let mockPlugin: any;
+  const mockRegion = Region.EU;
+  const mockUser: User = {
+    ovhSubsidiary: 'FR',
+    email: 'test@example.com',
+  } as User;
+
+  beforeEach(() => {
+    vi.clearAllMocks();
+    mockPlugin = {
+      configureTracking: mockConfigureTracking,
+      trackPage: vi.fn(),
+      trackClick: vi.fn(),
+    };
+    (TrackingPlugin as any).mockImplementation(() => mockPlugin);
+  });
+
+  it('should render DataUsagePolicy component', () => {
+    render(
+      <TrackingProvider region={mockRegion}>
+        <div>Test</div>
+      </TrackingProvider>,
+    );
+
+    expect(screen.getByTestId('data-usage-policy')).toBeInTheDocument();
+  });
+
+  it('should initialize TrackingPlugin and configure tracking', async () => {
+    render(
+      <TrackingProvider region={mockRegion}>
+        <div>Test</div>
+      </TrackingProvider>,
+    );
+
+    expect(TrackingPlugin).toHaveBeenCalledTimes(1);
+
+    await waitFor(() => {
+      expect(mockPlugin.configureTracking).toHaveBeenCalledWith(
+        mockRegion,
+        expect.any(Object),
+        'fr_FR',
+      );
+    });
+  });
+
+  it('should provide trackClick, trackPage and setUser functions in context', () => {
+    let contextValue: any;
+
+    const TestComponent = () => {
+      contextValue = useTrackingContext();
+      return <div>Test</div>;
+    };
+
+    render(
+      <TrackingProvider region={mockRegion}>
+        <TestComponent />
+      </TrackingProvider>,
+    );
```

Comment:
Maybe you can optimise the extract context like 
```suggestion
   const extractContext = () => {
    let contextValue;
    const TestComponent = () => {
      contextValue = useTrackingContext();
      return <div>Test</div>;
    };
    renderWithTracking(<TestComponent />);
    return contextValue;
  };
```
and you can use : 
```
const ctx = extractContext();
```